.github/scripts/github_common.py

```diff
@@ -17,14 +17,14 @@
 def get_header(gh_token: str) -> Dict[str, str]:
     return {"Authorization": f"token {gh_token.strip()}", "Accept": "application/vnd.github+json"}
 
-def get_runners(gh_token: str) -> List:
+def get_runners(gh_token: str) -> List[Dict[str, Any]]:
```

Comment:
Is it possible to be more specific than Any? since we're pulling down a json from http, is it possible to get something that isn't either `int` or `string`? This is safer for sure.

Reply:
Technically the dict can have values of lists, strs, dicts, bools, floats, etc so its a bit more complicated. Looking around quickly there is a way to specify a JSON type manually (https://github.com/python/typing/issues/182) but I'm not sure where to put the alias definition (I think it would be good to avoid duplicating this in places). I figured this was sufficient since we do this in other places but we can make this an issue for future work.